integration/e2e_test.go

```diff
@@ -90,9 +99,14 @@ func TestSnapshotCacheSingleEnvoyAndXdsRelayServer(t *testing.T) {
 				}, 1*time.Second, 100*time.Millisecond).Should(gomega.Equal(nListeners))
 			}
 
+			// If envoy never starts, the buffer is empty
+			assert.NotEmpty(t, envoyLogsBuffer.String())
```

Comment:
This condition could have prevented the false positive.

Reply:
I don't understand how this helps. The envoy logs are only for debugging in case the tests fails.

Reply:
yes...additionally when envoy is not able to start up the logs are empty.